Userland/Libraries/LibX86/Instruction.h

```diff
@@ -213,17 +218,19 @@ struct InstructionDescriptor {
     bool has_rm { false };
     unsigned imm1_bytes { 0 };
     unsigned imm2_bytes { 0 };
+    bool long_mode_default_64 { false };
+    bool long_mode_force_64 { false };
```

Comment:
We dont expect to have any 32 bit binaries when using the x64 build, so you can elide any full backwards compatibility features

Confirmation by maintainer needed

Reply:
Yeah, at the moment disasm assumes it only gets ELF files for the platform it's running on. But maybe we want to make it take that information from the ELF header in the future?